tests/geometry/functional/test_so3.py

```diff
@@ -27,8 +27,7 @@ def test_exp(batch_size: int, dtype: torch.dtype):
 def test_adjoint(batch_size: int, dtype: torch.dtype):
     rng = torch.Generator()
     rng.manual_seed(0)
-    tangent_vector = torch.rand(batch_size, 3, dtype=dtype, generator=rng)
-    group = so3.exp(tangent_vector)
+    group = so3.rand(batch_size, generator=rng, dtype=dtype)
```

Comment:
We are testing `rand` indirectly as part of other tests which might be okay, but none of the tests use `randn` so it's currently not tested in CI.

Reply:
`rand()` are tested offline. The only difference is how to generate the tangent vector. So I think it is is safe.

Reply:
Offline testing is always nice. I am wondering though whether it is good to add to the unit tests as well so that CI checks it regularly.